test/e2e/virtual/workspaces/virtual_workspace_test.go

```diff
@@ -75,12 +77,60 @@ var testData = testDataType{
 	workspace2Disambiguited: &tenancyv1beta1.Workspace{ObjectMeta: metav1.ObjectMeta{Name: "workspace2--1"}},
 }
 
+func createOrgMemberRoleForGroup(ctx context.Context, kubeClient kubernetes.Interface, orgClusterName string, groupNames ...string) error {
```

Comment:
// TODO: move this into a controller

Reply:
fixed in commit 56947bcb00c6b1746f053060eb32042705a133a2